--- bin/jruby.sh.orig	2011-05-25 23:01:16.000000000 -0400
+++ bin/jruby.sh	2011-05-25 23:02:32.000000000 -0400
@@ -107,6 +107,13 @@ for j in "$JRUBY_HOME"/lib/jruby.jar "$J
     JRUBY_ALREADY_ADDED=true
 done
 
+j=$(build-classpath objectweb-asm/asm objectweb-asm/asm-util \
+    objectweb-asm/all objectweb-asm/asm-commons objectweb-asm/asm-analysis objectweb-asm/asm-tree \
+    bytelist constantine jline jcodings joni jna jnr-posix jaffl jffi joda-time \
+    yydebug nailgun emma jgrapht bsf jnr-netdb jruby yecht jruby-yecht )
+
+JRUBY_CP="$JRUBY_CP$CP_DELIMETER$j"
+
 
 # ----- Set Up The System Classpath -------------------------------------------
 
--- bin/jruby.bash.orig	2011-06-02 10:13:58.158447113 -0400
+++ bin/jruby.bash	2011-06-02 10:14:44.679447018 -0400
@@ -149,6 +149,14 @@ if $cygwin; then
     JRUBY_CP=`cygpath -p -w "$JRUBY_CP"`
 fi
 
+j=$(build-classpath objectweb-asm/asm objectweb-asm/asm-util \
+    objectweb-asm/all objectweb-asm/asm-commons objectweb-asm/asm-analysis objectweb-asm/asm-tree \
+    bytelist constantine jline jcodings joni jna jnr-posix jaffl jffi joda-time \
+    yydebug nailgun emma jgrapht bsf jnr-netdb jruby yecht jruby-yecht )
+
+JRUBY_CP="$JRUBY_CP$CP_DELIMETER$j"
+
+
 # ----- Set Up The System Classpath -------------------------------------------
 
 if [ "$JRUBY_PARENT_CLASSPATH" != "" ]; then
