diff -U 5 -r jruby-1.1.5.orig/src/org/jruby/platform/Errno.java jruby-1.1.5/src/org/jruby/platform/Errno.java
--- jruby-1.1.5.orig/src/org/jruby/platform/Errno.java	2008-11-03 14:40:58.000000000 -0800
+++ jruby-1.1.5/src/org/jruby/platform/Errno.java	2008-11-28 21:54:12.000000000 -0800
@@ -26,12 +26,12 @@
  * the terms of any one of the CPL, the GPL or the LGPL.
  ***** END LICENSE BLOCK *****/
 
 package org.jruby.platform;
 
-import com.kenai.constantine.Constant;
-import com.kenai.constantine.ConstantSet;
+import constantine.Constant;
+import constantine.ConstantSet;
 import java.lang.reflect.Field;
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Collections;
 import java.util.List;
diff -U 5 -r jruby-1.1.5.orig/src/org/jruby/Ruby.java jruby-1.1.5/src/org/jruby/Ruby.java
--- jruby-1.1.5.orig/src/org/jruby/Ruby.java	2008-11-03 14:41:15.000000000 -0800
+++ jruby-1.1.5/src/org/jruby/Ruby.java	2008-11-28 21:54:00.000000000 -0800
@@ -37,11 +37,11 @@
  * the provisions above, a recipient may use your version of this file under
  * the terms of any one of the CPL, the GPL or the LGPL.
  ***** END LICENSE BLOCK *****/
 package org.jruby;
 
-import com.kenai.constantine.Constant;
+import constantine.Constant;
 import java.io.ByteArrayInputStream;
 import java.io.FileDescriptor;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.PrintStream;
