--- build.xml	2008-02-10 04:57:50.000000000 -0800
+++ build.xml.patched	2008-03-02 02:27:42.000000000 -0800
@@ -39,12 +39,10 @@
   
   <patternset id="other.src.pattern">
     <include name="**/*.properties"/>
   </patternset>
   
-  <taskdef name="retro" classname="net.sourceforge.retroweaver.ant.RetroWeaverTask" classpathref="build.classpath"/>
-  
   <import file="netbeans-ant.xml" optional="true"/>
 
   <!-- Initializes the build -->
   <target name="init">
     <xmlproperty file="build-config.xml" keepRoot="false" collapseAttributes="true"/>
