--- bin/jruby.orig	2008-07-18 22:12:22.000000000 -0700
+++ bin/jruby	2008-07-29 19:55:28.000000000 -0700
@@ -67,10 +67,16 @@
         else
         JRUBY_CP="$j"
     fi
 done
 
+j=$(build-classpath objectweb-asm/asm objectweb-asm/asm-util \
+  objectweb-asm/asm-analysis objectweb-asm/asm-tree joni joda-time jline \
+  bytelist jna jna-posix jvyamlb jcodings constantine)
+
+JRUBY_CP="$JRUBY_CP$CP_DELIMETER$j"
+
 if $cygwin; then
     JRUBY_CP=`cygpath -p -w "$JRUBY_CP"`
 fi
 
 # ----- Set Up The System Classpath -------------------------------------------
