--- spec/java_integration/ant/task_spec.rb.orig	2013-01-04 18:19:00.000000000 +0100
+++ spec/java_integration/ant/task_spec.rb	2013-02-26 12:33:14.679301003 +0100
@@ -11,7 +11,7 @@
       property :name => "jar", :value => "spec-test.jar"
       property :name => "dir", :value => "build"
       taskdef :name => "jarjar", :classname => "com.tonicsystems.jarjar.JarJarTask",
-        :classpath => "${basedir}/build_lib/jarjar-1.0.jar"
+        :classpath => "${basedir}/build_lib/jarjar.jar"
 
       target :jar do
         jar :destfile => "${jar}", :compress => "true", :index => "true" do
