--- src/org/jruby/Ruby.java.orig	2011-05-25 23:09:35.000000000 -0400
+++ src/org/jruby/Ruby.java	2011-05-25 23:09:59.000000000 -0400
@@ -1447,7 +1447,7 @@ public final class Ruby {
         addLazyBuiltin("tempfile.rb", "tempfile", "org.jruby.libraries.TempfileLibrary");
         addLazyBuiltin("fcntl.rb", "fcntl", "org.jruby.libraries.FcntlLibrary");
         addLazyBuiltin("rubinius.jar", "rubinius", "org.jruby.ext.rubinius.RubiniusLibrary");
-        addLazyBuiltin("yecht.jar", "yecht", "YechtService");
+        addLazyBuiltin("yecht-ruby-0.0.2.jar", "yecht", "YechtService");
 
         if (is1_9()) {
             addLazyBuiltin("mathn/complex.jar", "mathn/complex", "org.jruby.ext.mathn.Complex");
